test/fixtures/prefixes.out.css

```diff
@@ -1,5 +1,6 @@
-
 body {
   -webkit-transition: -webkit-transform 1s;
+  transition: -webkit-transform 1s;
   transition: transform 1s;
+  transition: transform 1s, -webkit-transform 1s;
 }
```

Comment:
Yeah this is weird. I just took the output of `autoprefixer` but that doesn't seem correct...